### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by abhie06**

### CODE REVIEW

**Comment by abhi1918** [comment]
Isn't there any advanced classification techniques like Convolutional neural network available for classification?

**Comment by Laiba2511** [comment]
What is the accuracy of all the algorithms?

**Comment by abhie06** [comment]
> 
Yes, we will provide it but accuracy is similar

**Comment by abhie06** [comment]
> What is the accuracy of all the algorithms?
Most algorithms have similar accuracy but decision tree gives the highest accuracy

**Comment by abhie06** [comment]
> Isn't there any advanced classification techniques like Convolutional neural network available for classification?

No but we will provide it to test it's accuracy against basic classification techniques

**Comment by abhie06** [comment]
Is there any suggestions?

File: code.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    df['Prediction'] = df[['Close']].shift(-future_days)
+    df.tail(4)
+
+    #X
+    X = np.array(df.drop(['Prediction'], 1))[:-future_days]
+    #Y
+    y = np.array(df['Prediction'])[:-future_days]
+
+    x_train, x_test, y_train, y_test = train_test_split(X, y, test_size=0.25)
+
```
Comment (SumitJ3na) [inline code.py:289]: You are using the variable X in valid = df[X.shape[0]:], but it's not defined anywhere in the given code. I guess you should use X_train.  And also the predictions array is being assigned to the valid DataFrame's 'Predictions' column through the line valid['Predictions'] = predictions. Assigning values to a new column in a slice may not function as intended because valid is produced as a slice of the original DataFrame df. You can make a new DataFrame just for the forecasts to get around this problem.
